tools/depends/target/Makefile

```diff
@@ -5,17 +5,59 @@ ifneq ($(shell test -f $(PREFIX)/share/config.site && echo 1),1)
 endif
 
 DEPENDS = \
-	pcre expat gettext sqlite3 libgpg-error \
-	libgcrypt bzip2 libfstrcmp liblzo2 freetype2-noharfbuzz fontconfig \
-	openssl gmp nettle gnutls googletest curl nghttp2 \
-	libjpeg-turbo libpng fribidi harfbuzz libass \
-	libxml2 rapidjson libmicrohttpd mariadb libffi \
-	python3 libshairplay libfmt libspdlog \
-	libplist libcec libbluray tinyxml \
-	taglib libusb libnfs freetype2 \
-	pythonmodule-pil pythonmodule-pycryptodome pythonmodule-setuptools \
-	libxslt ffmpeg crossguid libudfread \
-	libdvdread libdvdnav libdvdcss p8-platform flatbuffers dav1d xz
+  bzip2 \
+  crossguid \
+  curl \
+  dav1d \
+  expat \
+  ffmpeg \
+  flatbuffers \
+  fontconfig \
+  freetype2 \
+  freetype2-noharfbuzz \
+  gettext \
+  gmp \
+  gnutls \
+  googletest \
+  harfbuzz \
+  libass \
+  libbluray \
+  libcec \
+  libdvdcss \
+  libdvdnav \
+  libdvdread \
+  libffi \
+  libfmt \
+  libfstrcmp \
+  libgcrypt \
+  libgpg-error \
+  libjpeg-turbo \
+  liblzo2 \
+  libmicrohttpd \
+  libnfs \
+  libplist \
+  libpng fribidi \
+  libshairplay \
+  libspdlog \
+  libudfread \
+  libusb \
+  libxml2 \
+  libxslt  \
```

Comment:
```suggestion
  libxslt \
```

Reply:
Just as an fyi @Rechi im looking to piggy back these suggestions in with https://github.com/xbmc/xbmc/pull/20381

Reply:
thanks for cleaning up